packages/components/src/combobox-control/index.js

```diff
@@ -247,11 +247,6 @@ function ComboboxControl( {
 								instanceId={ instanceId }
 								ref={ inputContainer }
 								value={ isExpanded ? inputValue : currentLabel }
-								aria-label={
-									currentLabel
-										? `${ currentLabel }, ${ label }`
-										: null
-								}
```

Comment:
For now, I decided to delete those lines like in the original PR. See https://github.com/WordPress/gutenberg/pull/44347#discussion_r978615389 and https://github.com/WordPress/gutenberg/pull/44347#discussion_r981917387 for additional context and motivations for this change

(cc @tellthemachines )